src/managers/auth-manager.js

```diff
@@ -160,10 +160,10 @@ AuthManager.prototype = $.extend(AuthManager.prototype, {
         var handleSuccess = function (response) {
             //jshint camelcase: false
             //jscs:disable
-            token = response.access_token;
+            var authHeader = 'Bearer ' + token;
 
             var userInfo = decodeToken(token);
-            var userGroupOpts = $.extend(true, {}, adapterOptions, { success: $.noop, token: token });
+            var userGroupOpts = $.extend(true, {}, adapterOptions, { userId: userInfo.user_id, success: $.noop, headers: { Authorization: authHeader } });
```

Comment:
Looks like https://github.com/forio/epicenter-js-libs/pull/75/files covers this change; but, why do we need to pass in the userid here?